packages/charts/src/components/distributed-chart-tooltip/distributed-chart-tooltip.component.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { FC } from 'react';
+import { ScaleSettings, TooltipFormatter } from '@keen.io/charts-utils';
+
+import useDistributedChartTooltipSettings from '../../hooks/use-distributed-chart-tooltip-settings';
```

Comment:
Could You please simplify import to:
`import { useDistributedTooltipSettings }  from '../../hooks';`